### ISSUE TITLE
Frustrating Experience with Next.js: Endless Errors and Unmet Expectations

### ISSUE THREAD

**Issue by sunlin-xiaonai** [stackblitz/bolt.new#31]
### Describe the bug

mutiple language translate error

### Link to the Bolt URL that caused the error

https://bolt.new/~/sb1-k8hdyb

### Steps to reproduce

mutiple language translate error

### Expected behavior

mutiple language translate error

### Screen Recording / Screenshot

<img width="937" alt="image" src="https://github.com/user-attachments/assets/b5f2017a-5f38-4668-b881-5185ae3a29e3">

### Platform

web

### Additional context

I’ve encountered a continuous stream of errors while using the product, trying to fix them one after another, but the issues just won’t go away. I built a simple landing page using Next.js, and it’s been nothing but trouble—missing packages, unrecognized module names, and more. Despite all this, the performance that should be 100/100 barely reaches 50. I just topped up my account, but I’m already thinking about asking for a refund.

**Comment by codenomnom** [comment]
I've stumbled upon some errors myself - not exactly related to translations, but made by the AI itself. The reason was obvious, as the generated code wasn't perfect. It really depends on the prompts, as it sometimes tries to do very fancy stuff without being able to follow all the modifications required in all files. It seems to me it sometimes does what it needs to do in a file, then moves on and at because of the next "goal" - modifies the first file in a way that breaks the first feature.

I might be wrong. I'm just saying - it seems to me it's too early to rely on it coping with very complex tasks. Doing very small iterations like was working wonders in my case. Things like "setup translations using X library, preparing services and hooks without actually using them" is a good first step. Then "integrate into component Y" should not get you into trouble.

It requires patience and it's surely not perfect, and yet I think it's incredibly powerful and an excellent time saver if used carefully.

**Comment by sunlin-xiaonai** [comment]
yeah, i agree, now help me a lot , hope it better along with time
